content/src/filetype.js

```diff
@@ -184,11 +183,20 @@ function wrapTurtle(doc, domain, pragmasOnly, setupScript, instrumenter) {
       maintype = 'text/javascript';
     }
   }
-  var mainscript = '<script type="' + maintype + '">\n' + seeline;
+  var mainscript = '<script type="' + maintype + '">\n';
+  if (/javascript/.test(maintype)) {
+    // Wrap javascript in a closure (coffeescript does this automatically).
+    mainscript += '(function(){\n';
```

Comment:
Ah, I wasn't sure what we wanted to do about that. That makes sense, I'll fix it up to work that way.